src/components/Library/Library.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import SongCard from '../SongCard/SongCard';
+
+const Library = (props) => {
```

Comment:
maybe destructure here: { songs }

Reply:
I had thought about it but it was so short I figured it would be easier to just access it with dot notation